examples/fungible/tests/cross_chain.rs

```diff
@@ -95,7 +95,9 @@ async fn test_bouncing_tokens() {
     let receiver_chain = validator.new_chain().await;
     let receiver_account = AccountOwner::from(receiver_chain.public_key());
 
-    let messages = sender_chain
+    receiver_chain.register_application(application_id).await;
```

Comment:
Is this needed? If I understand correctly, it shouldn't affect anything because the message is rejected, so it shouldn't be executed?

Reply:
The problem is that we later call `fungible::query_account` to verify that the transfer hasn't arrived. That only works if the app is registered. But we could omit that check, or register the application only later, before that check.

Reply:
Oh, okay. I think it's okay to leave it as is then.